// @ludovicgueth/react-components/lib/Components/Textarea.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _react = require("react");

var _react2 = _interopRequireDefault(_react);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var Textarea = function Textarea(_ref) {
  var name = _ref.name,
      label = _ref.label,
      value = _ref.value,
      onChange = _ref.onChange,
      placeholder = _ref.placeholder,
      _ref$row = _ref.row,
      row = _ref$row === undefined ? 3 : _ref$row,
      className = _ref.className,
      help = _ref.help,
      helpValue = _ref.helpValue;

  return _react2.default.createElement(
    "div",
    { className: "field" },
    _react2.default.createElement(
      "label",
      { className: "label" },
      label
    ),
    _react2.default.createElement(
      "div",
      { className: "control" },
      _react2.default.createElement("textarea", {
        name: name,
        className: "textarea " + className,
        placeholder: placeholder,
        value: value,
        onChange: onChange,
        rows: row
      })
    ),
    help && _react2.default.createElement(
      "p",
      { className: "help " + className },
      helpValue
    )
  );
};

exports.default = Textarea;
