// @ludovicgueth/react-components/src/Components/Input.js
import React from "react";

import PropTypes from "prop-types";

const Input = ({
  name,
  label,
  type,
  placeholder,
  className,
  value,
  onChange,
  help,
  helpValue
}) => {
  return (
    <div className="field">
      <label className="label">{label}</label>
      <div className="control">
        <input
          className={"input " + className}
          type={type}
          placeholder={placeholder}
          name={name}
          value={value}
          onChange={onChange}
        />
      </div>
      {help && <p className={"help " + className}>{helpValue}</p>}
    </div>
  );
};

Input.defaultProps = {
  name: "name",
  label: "label",
  type: "text",
  placeholder: "",
  className: "",
  help: false,
  helpValue: ""
};

Input.propTypes = {
  name: PropTypes.string,
  label: PropTypes.string,
  type: PropTypes.string,
  placeholder: PropTypes.string,
  className: PropTypes.string,
  help: PropTypes.bool,
  helpValue: PropTypes.string
};

export default Input;

{
  /* const Input = ({
  name,
  label,
  type,
  value,
  onChange,
  placeholder,
  onlyInput,
  isDanger = false
}) => {
  if (onlyInput) {
    return (
      <input
        name={name}
        className={isDanger ? "input is-danger" : "input"}
        type={type}
        value={value}
        onChange={onChange}
        placeholder={placeholder}
      />
    );
  }
  return (
    <div className="field">
      <label className="label">{label}</label>
      <div className="control">
        <input
          name={name}
          className={isDanger ? "input is-danger" : "input"}
          type={type}
          value={value}
          onChange={onChange}
          placeholder={placeholder}
        />
      </div>
    </div>
  );
};

export default Input; */
}
